content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
-  * [Modifier Extension](#modifier-extensions){:target="_blank"}
-     * [Condition Result](#custom-extensions){:target="_blank"}
-     * [Condition Lifecycle Status](#extensions){:target="_blank"}
-  * [Extension](#extensions){:target="_blank"}
-     * [Condition Course](#extensions){:target="_blank"}
-     * [Family Member History Severity](#extensions){:target="_blank"}
+  * [Modifier Extension](https://hl7.org/fhir/r4/domainresource-definitions.html#DomainResource.modifierExtension){:target="_blank"}
+     * [Condition Result](https://hl7.org/fhir/r4/domainresource-definitions.html#DomainResource.modifierExtension){:target="_blank"}
+     * [Condition Lifecycle Status](https://hl7.org/fhir/r4/domainresource-definitions.html#DomainResource.modifierExtension){:target="_blank"}
+  * [Extension](https://hl7.org/fhir/r4/domainresource-definitions.html#DomainResource.extension){:target="_blank"}
+     * [Condition Course](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
+     * [Family Member History Severity](https://terminology.hl7.org/CodeSystem-v3-RoleCode.html){:target="_blank"}
 
 
 ## Terminology Bindings
 
 <%= terminology_table(:family_member_history, :r4) %>
 
-## Modifier Extensions
-
-* [Condition Result]
-* [Condition Lifecycle Status]
-
-## Extensions
-
-* [Patient Adopted]
-* [Precision Extension]
-* [Condition Course]
-* [Family Member History Severity]
-
 ## Custom Modifier Extensions and Extensions
 
-URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{ID}`
 
 ### Modifier Extensions
 
 ID              | Value\[x] Type                                                    | Description
 -----------------------------|-----------------------------------------------------------------------------|-----------------------------------------------
-`condition-result`           | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the presence (positive) or absence (negative) of a given condition.
-`condition-lifecycle-status` | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of whether a condition is active, inactive, resolved, etc.
+`condition-result`           | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the presence (positive) or absence (negative) of a given condition.
+`condition-lifecycle-status` | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of whether a condition is active, inactive, resolved, etc.
 
 
 ### Extensions
 
 ID              | Value\[x] Type                                                    | Description
 -----------------------------|-----------------------------------------------------------------------------|-----------------------------------------------
```

Comment:
@cldeitrich, I can't tell why these are in the order that they are. do you know why or if we should just make them alphabetical? seems like the HL7 site has them alphabetical(-ish)